### PR TITLE
Stylings added to the calculator

### PR DESCRIPTION
**Pull request by expjazz**
As required, the styles were added to all the components. Since it was not specified in the requirements how I should add the styles, I decided to use the Styled Components so I could get practice with them.

image

### CODE REVIEW

File: src/components/ButtonPannel.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      <div className="row rowFour">
+        <Button name="1" />
+        <Button name="2" />
+        <Button name="3" />
+        <Button name="+" />
+      </div>
+      <div className="row rowFive">
+        <Button name="0" wide />
+        <Button name="." />
+        <Button name="=" />
```
Comment (bellom) [inline src/components/ButtonPannel.js]: Great Job Developer;
Please you need to pass “color” props to the Button.
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896093/


In the end, you should have something like this 👇 ;
image

File: src/components/ButtonPannel.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import styled from 'styled-components';
+import Button from './Button';
+
+const StyledButtomPanel = styled.div`
+  .row {
+    height: 100px;
+    display: flex;
+    flex-grow: 1;
+  }
+
```
Comment (bellom) [inline src/components/ButtonPannel.js:11]: **[Optional]**
It is advisable to keep all CSS styling in a different file.
image

File: src/components/Button.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+const StyledButton = styled.button`
+width: ${props => (props.wide ? '50%' : '25%')};
+border: solid 1px black;
+background: ${props => props.color};
+p {
+  text-align: center;
+  color: black;
+}
+
+`;
```
Comment (bellom) [inline src/components/Button.js:14]: **[Optional]**
It is advisable to keep all CSS styling in a different file.
image

File: src/components/App.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+import ButtonPannel from './ButtonPannel';
+
+const StyledCalculator = styled.div`
+  width: 700px;
+  margin: auto;
+  display: "flex";
+  flex-direction: "column";
+  flex-wrap: "wrap";
+
+`;
```
Comment (bellom) [inline src/components/App.js:14]: **[Optional]**
It is advisable to keep all CSS styling in a different file.
image

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @expjazz  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by expjazz** [comment]
- As required, I passed the color props to the buttons.

- As of the styling, since I'm using styled-components, I'm not using a CSS file. Please let me know if this is not the correct way of using styled-components, or if it is not allowed / good practice.

Thanks for the review :raised_hands:

**Review by fatymahmed — APPROVED** [review]
## Hi @expjazz,
Good job implementing the changes 🎉🎉
## Status: Approved ✔️
_You may go ahead and merge this PR_

**Pull request closed without merging**